driver-core/src/test/functional/com/mongodb/internal/operation/FindAndDeleteOperationSpecification.groovy

```diff
@@ -296,23 +296,23 @@ class FindAndDeleteOperationSpecification extends OperationFunctionalSpecificati
         def originalException = new MongoSocketException('Some failure', new ServerAddress())
 
         when:
-        testRetryableOperationThrowsOriginalError(operation, [[3, 6, 0], [3, 4, 0]],
+        testRetryableOperationThrowsOriginalError(operation, [[3, 6, 0], [3, 6, 0], [3, 4, 0]],
```

Comment:
Added an extra server version due to the extra `connection.getDescription().getMaxWireVersion()` checks.